redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1431 @@
+                        self.bot,
+                        after.guild,
+                        datetime.now(timezone.utc),
+                        "cunmute",
+                        user,
+                        None,
+                        _("Manually removed channel overwrites"),
+                        until=None,
+                        channel=after,
+                    )
+                    log.debug("created case")
+            if to_del:
+                for u_id in to_del:
+                    del self._channel_mutes[after.id][u_id]
+                await self.config.channel(after).muted_users.set(self._channel_mutes[after.id])
+
+    @commands.Cog.listener()
+    async def on_member_join(self, member: discord.Member):
+        guild = member.guild
+        if await self.bot.cog_disabled_in_guild(self, guild):
+            return
+        mute_role = await self.config.guild(guild).mute_role()
+        if not mute_role:
+            return
+        if guild.id in self._server_mutes:
+            if member.id in self._server_mutes[guild.id]:
+                role = guild.get_role(mute_role)
+                if not role:
+                    return
+                until = self._server_mutes[guild.id][member.id]["until"]
+                await self.mute_user(
+                    guild, guild.me, member, until, _("Previously muted in this server.")
+                )
+
+    @commands.group()
+    @commands.guild_only()
+    async def muteset(self, ctx: commands.Context):
+        """Mute settings."""
+        pass
+
+    @muteset.command(name="forcerole")
+    @commands.is_owner()
+    async def force_role_mutes(self, ctx: commands.Context, force_role_mutes: bool):
+        """
+        Whether or not to force role only mutes on the bot
+        """
+        await self.config.force_role_mutes.set(force_role_mutes)
+        if force_role_mutes:
+            await ctx.send(_("Okay I will enforce role mutes before muting users."))
+        else:
+            await ctx.send(_("Okay I will allow channel overwrites for muting users."))
+
+    @muteset.command(name="settings", aliases=["showsettings"])
+    @checks.mod_or_permissions(manage_channels=True)
+    async def show_mutes_settings(self, ctx: commands.Context):
+        """
+        Shows the current mute settings for this guild.
+        """
+        data = await self.config.guild(ctx.guild).all()
+
+        mute_role = ctx.guild.get_role(data["mute_role"])
+        notification_channel = ctx.guild.get_channel(data["notification_channel"])
+        default_time = timedelta(seconds=data["default_time"])
+        msg = _(
+            "Mute Role: {role}\nNotification Channel: {channel}\n" "Default Time: {time}"
+        ).format(
+            role=mute_role.mention if mute_role else _("None"),
+            channel=notification_channel.mention if notification_channel else _("None"),
+            time=humanize_timedelta(default_time) if default_time else _("None"),
```

Comment:
Argument to `humanize_timedelta()` are keyword-only.
```suggestion
            time=humanize_timedelta(timedelta=default_time) if default_time else _("None"),
```
Alternatively you could just pass seconds directly to `humanize_timedelta()` and ditch the `default_time` variable.